app/config/sources.inc.php

```diff
@@ -408,12 +408,17 @@
     ],
     'mozorg/about/governance/policies/participation.lang' => [
         'supported_locales' => [
-            'de', 'es-ES', 'fr', 'hi-IN', 'ja', 'pt-BR', 'zh-TW',
+            'ar', 'de', 'es-ES', 'fr', 'hi', 'id', 'it', 'ja', 'nl', 'pl', 'pt-BR', 'ru', 'zh-CN', 'zh-TW',
```

Comment:
These pages are not opt-in, and I assume the list of locales is always going to be the same for all of them?

If that's the case, to make this less error-prone, you can also define an array at the top of the file, and only reference the array here ([example](https://github.com/mozilla-l10n/langchecker/blob/master/app/config/sources.inc.php#L29)).

Reply:
That's true, the same locales for all three pages

Reply:
Updated!